README.rst

```diff
@@ -171,7 +171,7 @@ The Go rules are tested and supported on the following host platforms:
 Users have reported success on several other platforms, but the rules are
 only tested on those listed above.
 
-Note: The latest version of these rules (v0.25.1) requires Bazel ≥ 3.4.0 to work.
+Note: The latest version of these rules (v0.25.1) requires Bazel ≥ 3.5.0 to work.
```

Comment:
No need to update the README yet. v0.25.x versions still work on Bazel 3.4.0. I'll update this as part of the next minor version release.

Reply:
I have reverted the comment.